app/src/main/AndroidManifest.xml

```diff
@@ -110,7 +110,7 @@
 
         <provider
             android:name="androidx.core.content.FileProvider"
-            android:authorities="chat.rocket.android.fileprovider"
+            android:authorities="chat.viasat.android.fileprovider"
```

Comment:
What does this affect?  Our application ID is currently "chat.veranda.android".....  where did "chat.viasat.android" come from?

Reply:
This solves the problem of not being able to install widechat if RC is present or vice-versa. I randomly changed it to `viasat` and it worked without causing any problems. So, I didn't give it much thought :sweat_smile: Should I change it to `veranda`?

Reply:
yep.  Please change to `veranda`.